### PR TITLE
[proto-render-setup Part 2]: Formal template rendering logic with sample template

### PR DESCRIPTION
**Pull request by Marcial1234**
Part of #247, explanation extended from #270 

Test on next PR, #272

### CODE REVIEW

File: third_party/render_md.go [hunk 1]
```diff
@@ -5,6 +5,13 @@ import (
 	"text/template"
 )
 
+func repeatedHeading(level int32) string {
+	return strings.Repeat("#", int(level))
+}
+
 func init() {
+	funcMap := template.FuncMap{
```
Comment (Marcial1234) [inline third_party/render_md.go]: ignore not being used.

File: third_party/render_globals.go [hunk 2]
```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"log"
 	"text/template"
+	"github.com/googlecodelabs/tools/claat/util"
```
Comment (shawnbuso) [inline third_party/render_globals.go]: Alphabetical order for imports

Comment (Marcial1234) [reply third_party/render_globals.go]: Per other files in this repo, imports follow alphabetical order, but standard library imports and our own packages are separate. Missed a new line here

**Comment by Marcial1234** [comment]
Hold on further reviewing until Design Doc is approved, closing for now

**Pull request closed without merging**